Update Glossary for 16.1

corinnaguenther opened the pull request:
[#63187] Update Glossary with new terms for 16.1

https://community.openproject.org/work_packages/63187

Changes:
- added #Phase
- added #Phase gate
- added #Project life cycle
- added #PM2 (both writings for SEO reasons)
- added #Lag
- added #Reminder
- and some minor adjustments in other sections.

corinnaguenther committed:
Update README.md

[#63187] Update Glossary with new terms for 16.1

https://community.openproject.org/work_packages/63187

Changes:
- added #Phase
- added #Phase gate
- added #Project life cycle
- added #PM2 (both writings for SEO reasons)
- added #Lag
- and some minor adjustments in other sections.

corinnaguenther committed:
Added #Reminder

MayaBerd commented on docs/glossary/README.md:
```diff
@@ -290,19 +298,31 @@ OpenID Connect (OIDC) is an authentication protocol built on OAuth 2.0 that enab
 
-A phase in OpenProject is defined as a [work package type](#work-package-types), which usually includes several work packages of types like task or feature. For example, typical phases for a construction repair project would be the following: Project definition, detailed Design & Tender, Construction and Post project.
+A phase in OpenProject can either be a [work package type](#work-package-types) or [project](#project) phase.
+
```
I think I would actually split Phase (wp type) and Project phase into two terms. This way, when we link to a glossary term, we use the one that is meant in a specific context.

corinnaguenther replied:
@MayaBerd I was thinking about this as well. I like the way you suggested it and will do it that way. Because if people search for the project phase, they might not use the whole term.

MayaBerd commented on docs/glossary/README.md:
```diff
@@ -290,19 +298,31 @@ OpenID Connect (OIDC) is an authentication protocol built on OAuth 2.0 that enab
+
+ The **type 'Phase'** usually includes several work packages of types like task or feature. For example, typical phases for a construction repair project would be the following: Project definition, detailed Design & Tender, Construction and Post project.
+
+ The **project phase** is part of the overall [project life cycle](#project-life-cycle) and helps structure the project's progression over time. A project phase can begin or end with a [phase gate](#phase-gate), marking key decision points. In OpenProject, administrators can enable some or all of the following project phases: Initiating, Planning, Executing, and Closing. These phases are based on ...
```
Should we mention that it is possible to add other project phases as an Enterprise add-on?

MayaBerd commented on docs/glossary/README.md:
```diff
@@ -367,6 +395,12 @@ The quick context menu in OpenProject opens when you open a [work package table]
+
+OpenProject offers different types of reminders so that you can lean back and never forget a task. One is the [date alert](#date-alerts) (Enterprise add-on), which generates automatic and customized [notifications](#notifications) regarding a work package's due date or start date.
+
+OpenProject also allows you to seet **work package reminders**:  Simply activate the clock icon on top of a work package and choose from different options, e.g. to be reminded the next day (at 9 am) or in one week. You can also set a custom date. [Read more about work package reminders in OpenProject](../user-guide/work-packages/#work-package-reminders).
```
```suggestion
OpenProject also allows you to set **work package reminders**:  Simply activate the clock icon on top of a work package and choose from different options, e.g. to be reminded the next day (at 9 am) or in one week. You can also set a custom date. [Read more about work package reminders in OpenProject](../user-guide/work-packages/edit-work-package/#work-package-reminders).
```

MayaBerd reviewed (commented):
@corinnaguenther I've added my comments

corinnaguenther committed:
Update docs/glossary/README.md

Co-authored-by: Maya Berdygylyjova <MayaBerd@users.noreply.github.com>

corinnaguenther committed:
update after review

- add #project-phase
- add internal links
- change + shorten #phase
- minor changes

corinnaguenther committed:
update after review

- add enterprise add-on hint to #project-phase

corinnaguenther committed:
Fix broken links

corinnaguenther committed:
fix broken link
